docs/strategy.md

```diff
@@ -172,6 +173,28 @@ Further readings:
 - [OPA UA explained in 1 minute (movie)](https://www.youtube.com/watch?v=-tDGzwsBokY)
 - [Industrial Internet Vocabulary](https://www.iiconsortium.org/vocab/index.htm)
 
+## <a id="api-security"></a>API Security
+Equinor APIs should be crafted with high emphasis on security. Code quality is a key foundation for API security. Like other software, APIs should be built by following recognized practices for software development. Examples of such practices includes code review, test driven development (TDD), automated testing, technical debt management & refactoring, etc.
+
+### Security tools and practices
+Development teams should include practices and tools addressing security issues in their development process. Automatic vulnerability scanning and keeping dependencies updated accordingly, is strongly recommended. Teams should stay updated on [OWASP API Security Top 10](https://github.com/OWASP/API-Security), and do regular evaluation on how these security risks are mitigated.
+
+API developers should understand and utilize the security features provided by the [API Gateway](#api-gateway), like throttling, IP restrictions and HTTP header validation.
+
```

Comment:
A few suggestions (on the borderline of a TR)(wording and location can be discussed). 

```suggestion
API's shall use encrypted transport. Transport layer security (TLS) is mandatory for REST based APIs
Input data to API's shall be sanitized.
```